lib/engine/game/g_18_cz.rb

```diff
@@ -87,6 +87,22 @@ class G18CZ < Base
         '8E' => :large,
       }.freeze
 
+      OFFBOARD_TILES_FOR_CORPORATION = {
+        'Ug' => '8894',
+        'kk' => '8895',
+        'SX' => '8896',
+        'PR' => '8897',
+        'BY' => '8898',
+      }.freeze
+
+      COORDINATES_FOR_LARGE_CORPORATION = {
```

Comment:
just to note that the [config already has the coords](https://github.com/tobymao/18xx/blob/master/lib/engine/config/game/g_18_cz.rb#L1136)